File: 백준/Bronze/29731. 2033년 밈 투표/2033년 밈 투표.cs
List<string> list = new List<string>()
        {
            "Never gonna give you up",
            "Never gonna let you down",
            "Never gonna run around and desert you",
            "Never gonna make you cry",
            "Never gonna say goodbye",
            "Never gonna tell a lie and hurt you",
            "Never gonna stop"
        };

        int cnt = 0;

        int input = int.Parse(Console.ReadLine());

        for (int i = 0; i < input; i++)
        {
            string str = Console.ReadLine();

            if (list.Contains(str))
                cnt++;
        }

        if (cnt == input)
            Console.WriteLine("No");
        else
            Console.WriteLine("Yes");

File: 백준/Bronze/29731. 2033년 밈 투표/2033년 밈 투표.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        List<String> list = new ArrayList<>();
        list.add("Never gonna give you up");
        list.add("Never gonna let you down");
        list.add("Never gonna run around and desert you");
        list.add("Never gonna make you cry");
        list.add("Never gonna say goodbye");
        list.add("Never gonna tell a lie and hurt you");
        list.add("Never gonna stop");

        int cnt = 0;

        Scanner scanner = new Scanner(System.in);
        int input = Integer.parseInt(scanner.nextLine());

        for (int i = 0; i < input; i++) {
            String str = scanner.nextLine();

            if (list.contains(str))
                cnt++;
        }

        if (cnt == input)
            System.out.println("No");
        else
            System.out.println("Yes");

        scanner.close();
    }
}
